src/kvirc/ui/KviMainWindow.cpp

```diff
@@ -981,6 +989,41 @@ void KviMainWindow::toggleStatusBar()
 	}
 }
 
+void KviMainWindow::toggleMenuBar()
+{
+	if(KVI_OPTION_BOOL(KviOption_boolMenuBarVisible) == true)
+	{
+		if(KVI_OPTION_BOOL(KviOption_boolWarnAboutHidingMenuBar))
+		{
+			// ah, we need an abstraction for this setCheckBox() qt5 shiny stuff
+			QMessageBox pMsgBox;
+			QCheckBox cb(__tr2qs("Do not show this message again"));
+			pMsgBox.setText(__tr2qs("This will hide the menu bar completely. "
```

Comment:
If the user is stupid enough to bypass this warning and then not know what has happened... In any case you can always ask him to press Ctrl+M.